test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -0,0 +1,1675 @@
+        value = (PBSNODES_ASJ_PATH, [PBSNODES_CMD, "-aSj"])
+        self.node_info[PBSNODES_ASJ_OUT] = value
+        value = (PBSNODES_AVS_PATH, [PBSNODES_CMD, "-avS"])
+        self.node_info[PBSNODES_AVS_OUT] = value
+        value = (PBSNODES_AS_PATH, [PBSNODES_CMD, "-aS"])
+        self.node_info[PBSNODES_AS_OUT] = value
+        value = (PBSNODES_AFDSV_PATH, [PBSNODES_CMD, "-aFdsv"])
+        self.node_info[PBSNODES_AFDSV_OUT] = value
+        value = (PBSNODES_AVFDSV_PATH, [PBSNODES_CMD, "-avFdsv"])
+        self.node_info[PBSNODES_AVFDSV_OUT] = value
+        value = (QMGR_PN_PATH, [QMGR_CMD, "-c", "p n @default"])
+        self.node_info[QMGR_PN_OUT] = value
+        value = (MOM_PRIV_PATH, None)
+        self.node_info[MOM_PRIV] = value
+        value = (MOM_LOGS_PATH, None)
+        self.node_info[MOM_LOGS] = value
+
+        # Comm information
+        value = (COMM_LOGS_PATH, None)
+        self.comm_info[COMM_LOGS] = value
+
+        # Hook information
+        value = (QMGR_PH_PATH, [QMGR_CMD, "-c", "p h @default"])
+        self.hook_info[QMGR_PH_OUT] = value
+        value = (QMGR_LPBSHOOK_PATH, [QMGR_CMD, "-c", "l pbshook"])
+        self.hook_info[QMGR_LPBSHOOK_OUT] = value
+
+        # Scheduler information
+        value = (QMGR_LSCHED_PATH, [QMGR_CMD, "-c", "l sched"])
+        self.sched_info[QMGR_LSCHED_OUT] = value
+        value = (SCHED_PRIV_PATH, None)
+        self.sched_info[SCHED_PRIV] = value
+        value = (SCHED_LOGS_PATH, None)
+        self.sched_info[SCHED_LOGS] = value
+
+        # Reservation information
+        value = (PBS_RSTAT_PATH, [PBS_RSTAT_CMD])
+        self.resv_info[PBS_RSTAT_OUT] = value
+        value = (PBS_RSTAT_F_PATH, [PBS_RSTAT_CMD, "-f"])
+        self.resv_info[PBS_RSTAT_F_OUT] = value
+
+        # Datastore information
+        value = (PG_LOGS_PATH, None)
+        self.datastore_info[PG_LOGS] = value
+
+        # PBS information
+        value = (PBS_CONF_PATH, None)
+        self.pbs_info[PBS_CONF] = value
+        value = (PBS_PROBE_PATH, [PBS_PROBE_CMD, "-v"])
+        self.pbs_info[PBS_PROBE_OUT] = value
+        # We'll append hostname to this later (see capture_pbs_info)
+        value = (PBS_HOSTN_PATH, [PBS_HOSTN_CMD, "-v"])
+        self.pbs_info[PBS_HOSTN_OUT] = value
+        value = (PBS_ENV_PATH, None)
+        self.pbs_info[PBS_ENVIRONMENT] = value
+
+        # Core file information
+        value = (CORE_SERVER_PATH, None)
+        self.core_info[CORE_SERVER] = value
+        value = (CORE_SCHED_PATH, None)
+        self.core_info[CORE_SCHED] = value
+        value = (CORE_MOM_PATH, None)
+        self.core_info[CORE_MOM] = value
+
+        # System information
+        value = (OS_PATH, None)
+        self.sys_info[OS_INFO] = value
+        value = (PROCESS_PATH, ["ps", "aux", "|", "grep", "pbs"])
+        self.sys_info[PROCESS_INFO] = value
+        value = (ETC_HOSTS_PATH, ["cat", os.path.join("/", "etc", "hosts")])
```

Comment:
Just to be consistent, can you use os.path.sep instead of "/" ?  I understand that windows /etc/pbs.conf in a different location, but let's at least be consistent.  If what you have there is run on windows, it'll end up with /\\etc\\hosts which is wrong.